wp-content/themes/cf-voyager/lessons-cpt.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Template Name: Lessons CPT
+ *
+ */
+
+get_header();
+
+$custom_terms = get_terms('lesson_types');
+
+foreach ($custom_terms as $custom_term) {
+	wp_reset_query();
+	$args = array(
+		'post_type' => 'cpt_lessons',
+		'posts_per_page' => 5,
+		'post_status' => 'publish',
+		'tax_query' => array(
+			array(
+				'taxonomy' => 'lesson_types',
+				'field' => 'slug',
+				'terms' => $custom_term->slug,
+			),
+		),
+	);
+}
+
+$the_query = new WP_Query($args);
+
+?>
+
+<?php
+if ($the_query->have_posts()) : ?>
+	<?php
+	echo esc_html($custom_term->name); ?>
+	<ul>
+		<?php
+		while ($the_query->have_posts()) : ?>
+			<?php
+			$the_query->the_post(); ?>
+
+			<?php
+			if (get_post_meta($post->ID, 'promoted_checkbox', true)) { ?>
+				<li>
+					<a href="<?php
+					the_permalink(); ?>">
+						<?php
+						the_title(); ?>
+					</a>
+				</li>
+				<?php
+			} else { ?>
```

Comment:
Whenever if/else, foreach are used in a template file, for larger code blocks, it is better to use the alternate syntax, as it is easier to read. See https://www.php.net/manual/en/control-structures.alternative-syntax.php)
So use:
```
<?php if (condition) : ?>
  ...
<?php else : ?>
...
<?php endif; ?>
```